app/views/organizations/_header.html.erb

```diff
@@ -36,30 +36,30 @@
         <div class="profile-header__meta">
           <% if @organization.location.present? %>
             <span class="profile-header__meta__item">
-              <%= inline_svg_tag("location.svg", aria: true, class: "crayons-icon mr-2 shrink-0", title: t("views.organizations.location.icon")) %>
+              <%= crayons_icon_tag(:location, css_class: "mr-2 shrink-0", title: t("views.organizations.location.icon")) %>
```

Comment:
Reading note for the rest of this PR: the class "crayons-icon" is always present when using `crayons_icon_tag` so it's missing from `css_class`. `aria: true` is also standard with the helper.